src/styles/blocks/page.scss

```diff
@@ -0,0 +1,10 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  &__body {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: $dayFontSize;
```

Comment:
<img width="332" alt="Screenshot 2023-10-29 at 16 48 15" src="https://github.com/mate-academy/layout_calendar/assets/135232001/4f9fa5ab-185c-45ff-a1ad-8cec2f4f9352">
Add these styles to your page to center your calendar vertically